python/dgl/readout.py

```diff
@@ -417,8 +417,9 @@ def broadcast_nodes(graph, graph_feat, *, ntype=None):
     broadcast_edges
     """
     if (F.shape(graph_feat)[0] != graph.batch_size and graph.batch_size == 1):
-        dgl_warning('If single graph, please make sure the batching dimension is added'
-                    ' and the tensor shape of graph_feat is (1, *).')
+        dgl_warning(':func:`broadcast_nodes` directly taking one node feature tensor as `feat[i]`'
```

Comment:
No need to have `:func:`. This only helps for rendering the documentation with sphinx.

Reply:
I see.. Thanks.